content/en/platform/corda/5.1/reference/corda-cli/preinstall.md

```diff
@@ -11,7 +11,11 @@ title: "preinstall"
 ---
 # preinstall
 This section lists the {{< tooltip >}}Corda CLI{{< /tooltip >}} `preinstall` arguments. You can use these commands to manually
-check Corda's configuration.
+check Corda's configuration and to confirm that all the Corda deployment [prerequisites]({{< relref "../../deploying-operating/deployment/prerequisites.md" >}})
```

Comment:
```suggestion
check Corda's configuration and to confirm that all the Corda deployment [prerequisites]({{< relref "../../deploying-operating/deployment/prerequisites.md" >}})
```

Should it be "deployment.... _is_", or "deployments.... _are_"?

Reply:
"... to confirm that all the Corda deployment [prerequisites] _are_ running..."

"are" refers to prerequisites